### PR TITLE
Addresses ticket #1965,  connection scheme setting update

### PR DESCRIPTION
**Pull request by DonatasKozlovskis**
Updated auto-connect feature by adding ability to choose connection scheme.

### CODE REVIEW

File: hawtio-web/src/main/webapp/app/jvm/js/connect.ts [hunk 1]
```diff
@@ -163,7 +163,7 @@ module JVM {
 	var autoconnect = $location.search();
 	if (typeof autoconnect != 'undefined' && typeof autoconnect.name != 'undefined') {	
 		var conOpts = Core.createConnectOptions({
-		        scheme: 'http',
+		        scheme: autoconnect.scheme,
```
Comment (davsclaus) [inline hawtio-web/src/main/webapp/app/jvm/js/connect.ts]: I wonder if we need to set http as fallback if the scheme is not configured on autoconnect? Or is it always configured?

**Comment by davsclaus** [comment]
Did you see my comment? Are you able to look into this?

**Comment by DonatasKozlovskis** [comment]
feature updated according your comment- http is set as fallback if the scheme is not configured on autoconnect

**Comment by davsclaus** [comment]
Thanks for coming back to this

**Pull request merged**